test/core/fling/BUILD

```diff
@@ -37,6 +37,11 @@ grpc_cc_binary(
     name = "fling_server",
     testonly = 1,
     srcs = ["server.cc"],
+    data = [
+        "//src/core/tsi/test_creds:ca.pem",
+        "//src/core/tsi/test_creds:server1.key",
+        "//src/core/tsi/test_creds:server1.pem",
+    ],
```

Comment:
Hardcoded paths like these won't import properly.

Reply:
Thanks for pointing this out! I refactored the code and it passed all the tests. Does this look good to you?
If path like this way doesn't meet the standard, we might need to refactor many of the previous clean-up PRs  since we merged a bunch of PRs with similar changes ... Can I know what might be the possible fix on this?

Reply:
Please make sure the path is changed during import.

Reply:
@nicolasnoble The import is fixed by @muxi with changes in copy.bara.sky file.

Reply:
I just manually checked internally and it looked like there were indeed some problems for importing like this. I updated the code and let's see how it is behaving.